Userland/Libraries/LibJS/Bytecode/Generator.h

```diff
@@ -241,6 +241,16 @@ class Generator {
     [[nodiscard]] size_t next_environment_variable_cache() { return m_next_environment_variable_cache++; }
     [[nodiscard]] size_t next_property_lookup_cache() { return m_next_property_lookup_cache++; }
 
+    [[nodiscard]] Operand add_constant(Value value)
```

Comment:
Ignore if we're not worried about atomicity for this PR, but `Operand` does not exist as of this commit